tests/phpstan-polyfill.php

```diff
@@ -4,7 +4,5 @@
 
 // PHPStan does not read global constants from the stubs yet, remove this when it does
 if (defined('OCI_NO_AUTO_COMMIT')) {
-    return;
+    define('OCI_NO_AUTO_COMMIT', 0);
 }
-
-define('OCI_NO_AUTO_COMMIT', 0);
```

Comment:
Sorry, I don't get it: to me it seems to do the opposite of what was done before… defining the variable when it is already defined.

Reply:
Wow… it should have defined the constant if it was _not_ defined. But given that it doesn't fail in its current state, we may try to remove this file completely.